src/pudl/transform/ferc1.py

```diff
@@ -2479,6 +2555,54 @@ def spot_fix_rows(self, df: pd.DataFrame) -> pd.DataFrame:
         return df
 
 
+class ElectricEnergyAccountSourcesTransformer(Ferc1AbstractTableTransformer):
+    """Transformer class for :ref:`electric_energy_account_sources_ferc1` table.
+
+    For XBRL, this is a duration-only table. Right now we are merging in the metadata
+    but not actually keeping anything from it. We are also not yet doing anything with
+    the sign.
+    """
+
+    table_id: Ferc1TableId = Ferc1TableId.ELECTRIC_ENERGY_ACCOUNT_SOURCES_FERC1
+    has_unique_record_ids: bool = False
+
+    def normalize_metadata_xbrl(
+        self, xbrl_fact_names: list[str] | None
+    ) -> pd.DataFrame:
+        """Normie.
+
+        Determine whether the
+        """
+        eeas_meta = (
+            super().normalize_metadata_xbrl(xbrl_fact_names)
+            # should this be a straight up rename? could we just rename it within the
+            # standard normalize_metadata_xbrl??
+            .assign(xbrl_column_stem=lambda x: x.xbrl_fact_name)
+        )
```

Comment:
not sure if the replace that is happening in the plant in service table is necessary here.